services/cell_logger/cell_logger.py

```diff
@@ -130,8 +138,14 @@ def write_to_db(timestamp, signal, cell_tech):
     
     if (signal != ''):
         
-        write_to_csv(timestamp, signal, cell_tech)
-        write_to_db(timestamp, signal, cell_tech)
+        if((log_env == 'CSV') or (log_env == 'CSV,DB') or (log_env  == 'DB,CSV')):
```

Comment:
And too repetitive. 

This could help:  https://pypi.org/project/environs/

Something like this might be best in the future: https://click.palletsprojects.com/en/7.x/options/#values-from-environment-variables

That allows config through both CLI and ENV. Could something like:

./cell_logger --csv <path> --postgres <connection_url>

AND

CSV=<path> POSTGRES=<connection_url> ./cell_logger

with the same code